### PR TITLE
Add named fonts

### PR DESCRIPTION
**Pull request by mfurseman**
Added the ability to specify a font by it's 'name' as stored in the CS-Studio fonts.def file. Updated corresponding tests.

### CODE REVIEW

**Comment by mfurseman** [comment]
@nickbattam

File: opimodel/fonts.py [hunk 1]
```diff
@@ -12,11 +12,15 @@
 
 class Font(object):
 
-    def __init__(self, fontface, size, style=REGULAR, pixels=True):
+    def __init__(self, name=None, fontface='Liberation Sans',
```
Comment (nickbattam) [inline opimodel/fonts.py:15]: Is this the right place to specify the default fontface? Can you just set the 'Default' font name instead? 

Having a default font defined here will potentially conflict with the Default defined in the CSS fonts.def file.

Comment (mfurseman) [reply opimodel/fonts.py:15]: I don't like that we have any named font face here, but it's not that much worse than specify 'Default' as that's a site specific preference anyway. We need to put some text string against  fontface in the xml. 

There shouldn't be any conflict as the font attributes are completely ignored if the named font is found.

Comment (mfurseman) [reply opimodel/fonts.py:15]: Perhaps 'Arial' is a better default font as it's very likely that every system will have that or a compatible fallback installed. Liberation Sans is Diamond's style guide approved default.

Comment (nickbattam) [reply opimodel/fonts.py:15]: The advantage of having a default of `name="Default"` is that will give the standard font in whatever environment it is used. 

If the fontface etc are ignored when a font 'name' is specified do you need default values for these too? Can you put placeholder or empty text in these properties? If you put specify `fontface='Arial'` and `name='Default'` your OPI file becomes confusing to anyone reading it as we'd see Liberation Sans.

Is there any way of parsing `fonts.def` to get the 'correct' default font?

**Pull request merged**